R/multitables.R

```diff
@@ -26,6 +26,60 @@ setMethod("[[", c("MultitableCatalog", "numeric"), function (x, i, ...) {
     getEntity(x, i, Multitable, ...)
 })
 
+#' @rdname catalog-extract
+#' @export
+setMethod("[[<-", c("MultitableCatalog", "character", "missing", "formula"),
+          function (x, i, j, value) {
+              stopifnot(length(i) == 1)
+              
+              ds <- loadDataset(datasetReference(x))
+              template <- formulaToQuery(value, data = ds)
+              
+              if (i %in% names(x)) {
+                  payload <- makeMultitablePayload(template = template)
+                  crPATCH(urls(x)[match(i, names(x))], body=toJSON(payload))
+                  return(x)
+              } else {
+                  ## Creating a new filter
+                  f <- .newMultitable(template, self(x), name = i)
+                  return(refresh(x))
+              }
+          })
+
+makeMultitablePayload <- function (template, name) {
+    template <- lapply(template$dimensions,
+                       function (x) list(query=x))
+
```

Comment:
Thanks for doing the TODO and removing the "variable" attribute of the template thing. Can you check the fixtures in inst and remove "variable" from them too accordingly?